spec/support/cohortManager.spec.js

```diff
@@ -0,0 +1,103 @@
+const CohortManager = require('../../src/cohortManager')
+
+describe('Test cohort manager', () => {
```

Comment:
You could maybe have a beforeEach in to reset before it runs each test - this might be giving some errors when data hasn't been cleared from your cohort between tests? (remember, they're ran in a random order each time)